backend/src/routes/metrics.ts

```diff
@@ -0,0 +1,48 @@
+/* Copyright Contributors to the Open Cluster Management project */
+
+import { Http2ServerRequest, Http2ServerResponse } from 'http2'
+import Prometheus, { register } from 'prom-client'
+import { logger } from '../lib/logger'
+
+interface PageData {
+  page: string
+}
+
+// register.setDefaultLabels({
+//   // label to identify customer?
+// })
+
+const acm_console_page_visit_counter = new Prometheus.Counter({
+  name: 'acm_console_page_visit_counter',
```

Comment:
Change "counter" to "count" to be consistent with other ACM metrics.
```suggestion
  name: 'acm_console_page_count',
```